bin/GameIndex.yaml

```diff
@@ -34907,8 +34859,6 @@ SLUS-20961:
     08901101:
       content: |-
         comment=Patch by Prafull
-        // New patch fixes most issues.
-        patch=0,EE,003a94f8,word,00000000
         // Fix hang at Level 4.
```

Comment:
Are the others lines of patch necessary?

Reply:
Yes, I confirmed with Prafull the other lines are still required

Reply:
Is the cause of the issue known?

Reply:
From my brief look at the debugger, it's generating a NaN float in the EE FPU and looping around comparing it.